README.md

```diff
@@ -8,15 +8,16 @@ The beginnings of a Sculpin-based style guide creation tool.
 ## Add Butler to a project
 1.   If the project does not already have npm dependencies run `npm init` to create a `package.json` file
 
-1.   Add Butler as a dependency by running `npm install --save git://github.com/palantirnet/butler.git#remove-ruby`
+1.   Add Butler as a dependency by running `npm install --save palantirnet/butler`
 
-## Getting Started with Butler
 1.  From inside the Butler root, run `npm install`
```

Comment:
I don't think we need to run npm install inside of butler anymore since is being installed already.